core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -275,5 +280,16 @@ oppia.controller('CollectionPlayer', [
         $scope.generatePathParameters();
       }
     }, true);
+
+    $scope.scrollToLocation = function(id) {
+      $location.hash(id);
+      $anchorScroll();
+    };
+    // Touching anywhere outside the mobile preview should hide it.
```

Comment:
@seanlip I know it sounds weird but it seems to work and makes sense when you look at the implementation. Like I mentioned earlier I've used `ng-if="!previewCardIsHidden"` (ie. when the card is not hidden) in both the exploration-summary-tile tags to show/hide the preview cards. So now the behavior doesn't seem flipped anymore in the mobile. 

@Oishikatta But changing so might flip the behavior in the desktop right?

Anyway, like @seanlip suggested I'll try to change it back to IsShown.